packages/db/src/models/file.ts

```diff
@@ -21,6 +21,16 @@ class FilePgModel extends BasePgModel<PostgresFile, PostgresFileRecord> {
       .onConflict(['bucket', 'key'])
       .merge();
   }
+
+  upsertById(
+    knexOrTrx: Knex | Knex.Transaction,
+    file: Partial<PostgresFileRecord>
```

Comment:
We should enforce that `cumulus_id` has to be property of the incoming `file` record here

Reply:
Hmm, it's not really an *upsert* then though.      

I can easily intersect a required key with that field, but perhaps the approach is wrong, and this really should be an update....    I suspect this was a victim of a refactor, originally I was updating postgres if the record didn't exist. 

Yes, I think that's better, and removes the upsert: c8d4942bd